src/node_metadata.h

```diff
@@ -37,7 +39,11 @@ namespace node {
   V(modules)                                                                   \
   V(nghttp2)                                                                   \
   V(napi)                                                                      \
-  V(llhttp)                                                                    \
+  V(llhttp)
+
+#if defined(UVWASI_VERSION_STRING)
```

Comment:
I don't think we need the `#if` because I don't think you can currently build Node without uvwasi. I would put this in `NODE_VERSIONS_KEYS_BASE`

Reply:
No need this variable anymore as we decided to avoid `#ifdef`.

Thanks for the above suggestions. PR in good shape now.